.github/workflows/build.yml

```diff
@@ -66,14 +67,49 @@ jobs:
             ${{ runner.os }}-gradle-
       - name: Build
```

Comment:
Let's name it PR Checks

Reply:
Note: we discussed this and decided to leave it unchanged